AVOutput/AVOutputTV.cpp

```diff
@@ -2889,9 +2889,16 @@ namespace Plugin {
         std::string pqmode;
         std::string source;
         std::string format;
-        int lowLatencyIndex = 0;
+        int params[3]={0};
```

Comment:
what are params here?  Can we give an appropriate name?

Reply:
These changes have been ported from the TVSettings plugin. The variable "params" is intended to hold the corresponding pqparams (brightness, contrast, lowlatency) values. However, it should be named "lowlatencyindex" since that name is already being used within this function.
We opted for a generic name like "params," and this value will be saved to the driver configuration file(ssmdata).

Currently, the generic name "params" is used in all set/reset functions. We can take this into account during the Llama AVOutput porting process.